### PR TITLE
branding: Add casp reporting links

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: templates/branding/openSUSE/external_reporting.html.ep [hunk 1]
```diff
@@ -35,9 +38,13 @@
 %         $product = join(' ', $flavor_to_prod_sle{$subproduct}, $version);
 %     }
 % }
-% else {
+% elsif ($job->DISTRI eq 'opensuse') {
 %     $product = $job->VERSION eq 'Tumbleweed' ? 'Tumbleweed' : 'Distribution';
 % }
+% elsif ($job->DISTRI eq 'casp') {
+%     # sic!
+%     $product = "1 .0\x{C2A0}(CASP/MicroOS)";
```
Comment (okurz) [inline templates/branding/openSUSE/external_reporting.html.ep]: the `\x{C2A0}` for non-breaking space does not really work, gives me a chinese hut instead. any ideas?

**Comment by okurz** [comment]
@kravciak

File: templates/branding/openSUSE/external_reporting.html.ep (hunk 1 above)
Comment (kraih) [inline templates/branding/openSUSE/external_reporting.html.ep]: You might be mixing up the encoded UTF-8 value with the Unicode code point.

Comment (kraih) [inline templates/branding/openSUSE/external_reporting.html.ep]: http://graphemica.com/%EC%8A%A0 and http://graphemica.com/00A0.

**Comment by okurz** [comment]
Great :) thx

**Pull request merged**